Build Chain failed for Java 11 and 17 when building kogito-runtimes project:
```[ERROR] Failed to execute commands for kiegroup/kogito-runtimes```

Error was:
```Detected Maven Version: 3.8.1 is not in the allowed range 3.8.6.```

I will push a new commit to update maven-version to 3.8.6 as it is required to build kogito-runtimes.

--- a/.github/workflows/full-downstream.yml
+++ b/.github/workflows/full-downstream.yml
@@ -25,7 +25,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         java-version: [11]
-        maven-version: ['3.8.1']
+        maven-version: ['3.8.6']
       fail-fast: false
     runs-on: ${{ matrix.os }}
     name: Full downstream build

--- a/.github/workflows/pull_request.yml
+++ b/.github/workflows/pull_request.yml
@@ -22,7 +22,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         java-version: [11, 17]
-        maven-version: ['3.8.1']
+        maven-version: ['3.8.6']
       fail-fast: false
     runs-on: ${{ matrix.os }}
     name: ${{ matrix.os }} - Java ${{ matrix.java-version }} - Maven

